src/Todo.css

```diff
@@ -1,20 +1,38 @@
-li {
-  background-color: white;
+.todo {
+  background-color: #ffffff;
   width: 80vw;
   padding: 25px 30px;
   margin: 25px 0;
-  box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
+  box-shadow: var(--regular-shadow);
   display: flex;
   align-items: center;
   justify-content: space-between;
+  border-left: 5px solid transparent;
 }
 
-li.done {
-  border-left: 5px solid #4caf50;
+.todo-checkbox{
+  transform: scale(1.2);
 }
 
-li i {
-  color: #f44336;
-  cursor: pointer;
-  font-size: 1.1rem;
+.todo-checkbox:checked{
+  accent-color: var(--primary-color);
```

Comment:
accent color change the style of our checkbox and should be given to  `.todo-checkbox` selector
css will handle different states of this checkbox.
move it to the mentioned selector if you agree with my reasoning.